config/locales/hyrax.en.yml

```diff
@@ -1058,6 +1061,9 @@ en:
         nestable: "NESTABLE"
         require_membership: "REQUIRE MEMBERSHIP"
         sharable: "SHARING"
+        share_applies_to_collection: "APPLY TO COLLECTION"
```

Comment:
Why are these all capitalized?  This seems to be different from all other labels in this file.